tools/topology/sof-cml-demux-rt5682.m4

```diff
@@ -15,17 +15,12 @@ include(`common/tlv.m4')
 include(`sof/tokens.m4')
 
 # Include Cannonlake DSP configuration
-include(`platform/intel/cnl.m4')
+ifelse(PLATFORM, `whl', include(`platform/intel/whl-sku.m4'),
+	ifelse(PLATFORM, `cml', include(`platform/intel/cml-sku.m4'), `'))
```

Comment:
Why not just do 
```
include(`platform/intel/PLATFORM.m4`)
```

Reply:
@lgirdwood PLATFORM is set from makefile. and sof-cml-demux-rt5682.m4 is shared between whl and cml. if/else is needed in somewhere doesn't it?
```
        "sof-cml-demux-rt5682\;sof-cml-demux-rt5682\;-DPLATFORM=cml"                                                                            
        "sof-cml-demux-rt5682\;sof-whl-demux-rt5682\;-DPLATFORM=whl"
```

Reply:
@lgirdwood now I got it. PLATFORM.m4. will update again.

Reply:
@lgirdwood I updated again, could you review agian?